src/js/components/MultiSelect/index.d.ts

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { DropProps } from '../Drop';
+import {
+  A11yTitleType,
+  AlignSelfType,
+  GridAreaType,
+  MarginType,
+  PlaceHolderType,
+} from '../../utils';
+
+export interface MultiSelectProps {
+  a11yTitle?: A11yTitleType;
```

Comment:
Similar to PropTypes can we pull out the identical Types used by both select and mutli-select and then just import them so we only have to go to one place to change for both?

Reply:
Good idea, I pull out identical props in the PropTypes and index.d.ts files